### ISSUE TITLE
Support Crieto Benchmark for Decision Forest Classification

### ISSUE THREAD

**Issue by jiazou-bigdata** [asu-cactus/netsdb#58]
https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary.html

**Comment by hguan6** [comment]
Thank you!

**Comment by hguan6** [comment]
With a V100 GPU, xgboost can train the whole training dataset in one day for a 1600-trees, depth-8 model. To train a 10-tree model takes only 23 minutes.
However, I found something strange, the labels of the test data set are all 0s and no 1s, while the training set has about 1/4 1s and 3/4 0s.
